daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/ValidationError.scala

```diff
@@ -515,3 +515,12 @@ final case class EWrongInterfaceRequirement(
   protected def prettyInternal: String =
     s"Interface $requiringIface does not require $wrongRequiredIface"
 }
+final case class ENotClosedInterfaceRequires(
```

Comment:
are we already tracking the tests we should add for this somewhere?

Reply:
I added an entry in #10917